HNC: make deploy at HEAD failed on a no-recent-change file

@GinnyJI and I both got this error when `make deploy` at HEAD:
```
go build -o "./bin/controller-gen" sigs.k8s.io/controller-tools/cmd/controller-gen
"./bin/controller-gen" object:headerFile=./hack/boilerplate.go.txt paths=./api/...
go fmt ./api/... ./cmd/... ./internal/... ./third_party/...
go vet ./api/... ./cmd/... ./internal/... ./third_party/...
# sigs.k8s.io/multi-tenancy/incubator/hnc/internal/reconcilers_test
internal/reconcilers/hierarchy_config_test.go:452:23: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
make: *** [Makefile:125: vet] Error 2
```
The line is
```
nm := createNS(ctx, string('a'+i))
```

Not sure what was recently changed that caused this issue.

--- a/incubator/hnc/internal/reconcilers/hierarchy_config_test.go
+++ b/incubator/hnc/internal/reconcilers/hierarchy_config_test.go
@@ -2,6 +2,7 @@ package reconcilers_test
 
 import (
 	"context"
+	"fmt"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
@@ -449,7 +450,7 @@ func hasChild(ctx context.Context, nm, cnm string) func() bool {
 func createNSes(ctx context.Context, num int) []string {
 	nms := []string{}
 	for i := 0; i < num; i++ {
-		nm := createNS(ctx, string('a'+i))
+		nm := createNS(ctx, fmt.Sprintf("%c", 'a'+i))
 		nms = append(nms, nm)
 	}
 	return nms

